app/views/storage/_storage_pod_folders.html.haml

```diff
@@ -5,7 +5,7 @@
       %tbody
         - @folders.each do |f|
           %tr{:title => _("View '%{name}' Cluster") % {:name => f.name},
-            :onclick => "miqTreeActivateNode('storage_pod_tree', 'dsc_xx-#{to_cid(f.id)}');"}
+            :onclick => "miqTreeActivateNode('storage_pod_tree', 'xx-#{to_cid(f.id)}');"}
```

Comment:
`to_cid()` here needs to go away, javascript won't understand the compressed id here and obviously,
the click won't work.